src/object/srv_obj_migrate.c

```diff
@@ -570,14 +571,190 @@ migrate_fetch_update_inline(struct migrate_one *mrone, daos_handle_t oh,
 	return rc;
 }
 
+static int
+obj_ec_encode_buf(daos_obj_id_t oid, struct daos_oclass_attr *oca,
+		  unsigned char *buffer, unsigned char *p_bufs[])
+{
+	struct obj_ec_codec	*codec;
+	unsigned int	cell_bytes = obj_ec_cell_rec_nr(oca);
```

Comment:
seems better to name it as cell_rec_nr   (cell_bytes used to mean cell_rec_nr * iod_size)

Reply:
right